src/NServiceBus.AcceptanceTests/Routing/MessageDrivenSubscriptions/When_subscribing_with_scaled_out_subscriber.cs

```diff
@@ -0,0 +1,77 @@
+namespace NServiceBus.AcceptanceTests.Routing.MessageDrivenSubscriptions
+{
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using EndpointTemplates;
+    using NUnit.Framework;
+    using Settings;
+
+    public class When_subscribing_with_scaled_out_subscriber : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public async Task Should_round_robin()
+        {
+            Requires.MessageDrivenPubSub();
+
+            var context = await Scenario.Define<Context>()
+                .WithEndpoint<Publisher>(b => b.When(async s =>
+                {
+                    await s.Publish(new MyEvent());
+                    await s.Publish(new MyEvent());
+                }))
+                .WithEndpoint<Subscriber>(b => b.CustomConfig(c => c.MakeInstanceUniquelyAddressable("1")))
+                .WithEndpoint<Subscriber>(b => b.CustomConfig(c => c.MakeInstanceUniquelyAddressable("2")))
+                .Done(c => c.HandledOnDistriminator.Count >= 2)
+                .Run();
+
+            Assert.That(context.HandledOnDistriminator, Does.Contain("1"));
+            Assert.That(context.HandledOnDistriminator, Does.Contain("2"));
+            Assert.That(context.HandledOnDistriminator.Count, Is.EqualTo(2));
+        }
+
+        class Context : ScenarioContext
+        {
+            public List<string> HandledOnDistriminator { get; } = new List<string>();
+        }
+
+        class Publisher : EndpointConfigurationBuilder
+        {
+            public Publisher()
+            {
+                EndpointSetup<DefaultServer>();
+            }
+        }
+
+        class Subscriber : EndpointConfigurationBuilder
+        {
+            public Subscriber()
+            {
+                EndpointSetup<DefaultServer>(c => { }, metadata => metadata.RegisterPublisherFor<MyEvent>(typeof(Publisher)));
```

Comment:
Should we have here `SubscribeWithInstanceSpecificQueue` option turned on? Otherwise the event ends in shared queue right?